docs/getting-started/rulesets.md

```diff
@@ -249,6 +249,51 @@ The example above will run all of the rules defined in the `spectral:oas` rulese
 
 - [Rules relevant to OpenAPI v2 and v3](../reference/openapi-rules.md)
 
+## Selectively silencing some results
+
+From time to time, you want to ignore some specific results without turning off 
+the rule entirely. This may happen, for instance, when working with legacy APIs.
+
+The ruleset can be extended to define such exceptions through the optional `except` property.
+
+`except` describes a map of locations (expressed as Json paths) and rules that should be ignored.
+
+Locations be can either described as relatively to the ruleset or absolute paths.
+
+```yaml
+extends: spectral:oas
+
+except:
+  "subfolder/one.yaml#"
+    - oas3-api-servers
+  "/tmp/docs/one.yaml#/info":
+    - info-contact
+    - info-description
+```
+
+If you want to pass your document to Spectral through `stdin`, the ruleset will 
+have to be expressed slightly differently, using a predefined `<STDIN>` pattern as
+the location.
+
+For instance, running the following command will honor the defined exceptions...
+
+```bash
+$ cat /path/to/the/document | spectral lint -r /path/to/ruleset.yaml
+```
+
+...provided the ruleset is defined this way:
+
+```yaml
+extends: spectral:oas
+
+except:
+  "<STDIN>#":
+    - oas3-api-servers
+  "<STDIN>#/info":
```

Comment:
Hm, not a fan of it tbh.
How about `./#info`? This would be more universal IMHO, as you could also target the input document, i.e. it'd target `foo.yaml` in when the following command `spectral lint foo.yaml` is executed.

Reply:
I've been recommending we just ditch STDIN support entirely it's not worth the hassle generally speaking we should plan features before we get people working on them :D 

This is trying to solve the Studio use case, and if people ask for this we can bring it in, but until then its scope creep.

Reply:
@P0lip @philsturgeon STDIN support has been dropped. A warning is now generated when the document is consumed from stdin and the ruleset contains exceptions.

Reply:
> This is trying to solve the Studio use case, and if people ask for this we can bring it in, but until then its scope creep.

Studio should not be affected too much, since it doesn't use STDIN.

Reply:
@P0lip that's what im trying to say. We are trying to get this working for the studio use case, which doesnt need STDIN, so we don't need to worry about STDIN here.